src/pages/FormPage.js

```diff
@@ -119,7 +119,7 @@ const Inputs = styled.div`
 `;
 
 const Map = styled.img`
```

Comment:
Maybe you could name the Map components after their page in order to avoid repeats in code.